### ISSUE TITLE
Enable `jupyter labextension uninstall .`

### ISSUE THREAD

**Issue by SimonBiggs** [jupyterlab/jupyterlab#4394]
It is currently possible to run `jupyter labextension install .` when in the directory of a labextension. It would be awesome to be able to also run `jupyter labextension uninstall .`.

Cheers :)

**Comment by ian-r-rose** [comment]
I agree @SimonBiggs. When working on extensions I often wish this would work.

**Comment by vidartf** [comment]
The relevant function is here: https://github.com/jupyterlab/jupyterlab/blob/93773c61de730fbcfb0834a39ade333c60ab606a/jupyterlab/commands.py#L537

I guess the change would entail to run `name` through [`_normalize_path`](https://github.com/jupyterlab/jupyterlab/blob/93773c61de730fbcfb0834a39ade333c60ab606a/jupyterlab/commands.py#L1292), and then check whether it is a directory, similar to here: https://github.com/jupyterlab/jupyterlab/blob/93773c61de730fbcfb0834a39ade333c60ab606a/jupyterlab/commands.py#L1171-L1172
